Build fails in CI on develop-v1 branch

```
yarn run v1.22.18
$ yarn workspace ts-graphviz build && yarn workspace @ts-graphviz/react build
$ rollup -c --configPlugin rollup-plugin-typescript2 --waitForBundleInput

./src/index.ts → ./lib/index.cjs, ./lib/index.js, ./lib/bundle.min.js...
created ./lib/index.cjs, ./lib/index.js, ./lib/bundle.min.js in 18.5s

./lib/index.d.ts → ./lib/index.d.ts...
created ./lib/index.d.ts in 293ms
$ prettier --write ./lib/index.*
lib/index.cjs 239ms
lib/index.d.ts 853ms
lib/index.js 120ms
$ rollup -c --configPlugin rollup-plugin-typescript2 --waitForBundleInput

./src/index.ts → ./lib/index.cjs, ./lib/index.js...
[!] (plugin rpt2) Error: /home/runner/work/ts-graphviz/ts-graphviz/packages/react/src/renderer/render.ts(2,23): semantic error TS230[7](https://github.com/ts-graphviz/ts-graphviz/runs/6435167935?check_suite_focus=true#step:8:7): Cannot find module 'ts-graphviz' or its corresponding type declarations.
src/renderer/render.ts
Error: /home/runner/work/ts-graphviz/ts-graphviz/packages/react/src/renderer/render.ts(2,23): semantic error TS2307: Cannot find module 'ts-graphviz' or its corresponding type declarations.
    at error (/home/runner/work/ts-graphviz/ts-graphviz/node_modules/rollup/dist/shared/rollup.js:19[8](https://github.com/ts-graphviz/ts-graphviz/runs/6435167935?check_suite_focus=true#step:8:8):30)
    at throwPluginError (/home/runner/work/ts-graphviz/ts-graphviz/node_modules/rollup/dist/shared/rollup.js:21[9](https://github.com/ts-graphviz/ts-graphviz/runs/6435167935?check_suite_focus=true#step:8:9)20:12)
    at Object.error (/home/runner/work/ts-graphviz/ts-graphviz/node_modules/rollup/dist/shared/rollup.js:22643:20)
    at Object.error (/home/runner/work/ts-graphviz/ts-graphviz/node_modules/rollup/dist/shared/rollup.js:22097:42)
    at RollupContext.error (/home/runner/work/ts-graphviz/ts-graphviz/node_modules/rollup-plugin-typescript2/src/rollupcontext.ts:37:18)
    at /home/runner/work/ts-graphviz/ts-graphviz/node_modules/rollup-plugin-typescript2/src/print-diagnostics.ts:41:11
    at arrayEach (/home/runner/work/ts-graphviz/ts-graphviz/node_modules/rollup-plugin-typescript2/node_modules/lodash/lodash.js:530:11)
    at Function._.each [as forEach] (/home/runner/work/ts-graphviz/ts-graphviz/node_modules/rollup-plugin-typescript2/node_modules/lodash/lodash.js:94[10](https://github.com/ts-graphviz/ts-graphviz/runs/6435167935?check_suite_focus=true#step:8:10):[14](https://github.com/ts-graphviz/ts-graphviz/runs/6435167935?check_suite_focus=true#step:8:14))
    at printDiagnostics (/home/runner/work/ts-graphviz/ts-graphviz/node_modules/rollup-plugin-typescript2/src/print-diagnostics.ts:9:2)
    at Object.transform (/home/runner/work/ts-graphviz/ts-graphviz/node_modules/rollup-plugin-typescript2/src/index.ts:244:5)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 1
Command: /opt/hostedtoolcache/node/14.[19](https://github.com/ts-graphviz/ts-graphviz/runs/6435167935?check_suite_focus=true#step:8:19).2/x64/bin/node
info Visit https://yarnpkg.com/en/docs/cli/workspace for documentation about this command.
Arguments: /usr/local/lib/node_modules/yarn/lib/cli.js build
Directory: /home/runner/work/ts-graphviz/ts-graphviz/packages/react
Output:

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/node-ci.yml
+++ b/.github/workflows/node-ci.yml
@@ -34,6 +34,7 @@ jobs:
       run: yarn test
       env:
         CI: 'true'
+        NODE_OPTIONS: --max-old-space-size=4096
     - name: Build
       run: yarn build
       env:

--- a/.github/workflows/node-ci.yml
+++ b/.github/workflows/node-ci.yml
@@ -18,6 +18,10 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - name: setup graphviz
       uses: ts-graphviz/setup-graphviz@v1
+    - name: Set Swap Space
+      uses: pierotofy/set-swap-space@master
+      with:
+        swap-size-gb: 10
     - name: Install dependencies
       run: yarn --frozen-lockfile --ignore-optional
       env:

--- a/package.json
+++ b/package.json
@@ -8,17 +8,11 @@
   ],
   "scripts": {
     "prebuild": "yarn workspace @ts-graphviz/parser build:peggy",
-    "build": "yarn workspace ts-graphviz build && yarn workspace @ts-graphviz/react build && yarn workspace @ts-graphviz/node build",
+    "build": "yarn workspace ts-graphviz build && yarn workspace @ts-graphviz/parser build && yarn workspace @ts-graphviz/react build && yarn workspace @ts-graphviz/node build",
     "test": "jest",
     "format": "eslint -c .eslintrc.json --ext ts,tsx . --fix && prettier --plugin=./config/prettier/peggy.json --write './**/*.(ts|tsx|peggy)'",
     "lint": "eslint -c .eslintrc.json --ext ts,tsx .",
     "doc": "typedoc"
-  },
-  "dependencies": {
-    "@ts-graphviz/node": "file:packages/node",
-    "@ts-graphviz/parser": "file:packages/parser",
-    "@ts-graphviz/react": "file:packages/react",
-    "ts-graphviz": "file:packages/ts-graphviz"
   },
   "devDependencies": {
     "@hpcc-js/wasm": "^1.14.1",

--- a/package.json
+++ b/package.json
@@ -7,7 +7,6 @@
     "packages/*"
   ],
   "scripts": {
-    "prebuild": "yarn workspace @ts-graphviz/parser build:peggy",
     "build": "yarn workspace ts-graphviz build && yarn workspace @ts-graphviz/parser build && yarn workspace @ts-graphviz/react build && yarn workspace @ts-graphviz/node build",
     "test": "jest",
     "format": "eslint -c .eslintrc.json --ext ts,tsx . --fix && prettier --plugin=./config/prettier/peggy.json --write './**/*.(ts|tsx|peggy)'",

